src/content-type/content-type.e2e-spec.ts

```diff
@@ -0,0 +1,54 @@
+import * as request from 'supertest';
+import { Test } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import { ContentTypeModule } from './content-type.module';
+
+describe('Cats', () => {
+  let app: INestApplication;
+
+  beforeAll(async () => {
+    const moduleRef = await Test.createTestingModule({
+      imports: [ContentTypeModule],
+    }).compile();
+
+    app = moduleRef.createNestApplication();
+    await app.init();
+  });
+
+  it(`can create collection`, () => {
+    request(app.getHttpServer())
+      .post('/content-type/collections/new-content-type')
+      .send({})
+      .expect(201);
```

Comment:
are there any params in the response payload to check?

Reply:
Return created object